### ISSUE TITLE
"No components found" box should be removed from Docs

### ISSUE THREAD

**Issue by CodeByAlex** [storybookjs/storybook#7451]
**Describe the bug**
When prop definitions arent set a box shows up saying "no component found". This makes sense if it was a requirement to show props but some frameworks arent supported for that feature in Docs yet. Therefore, I feel as though it should be removed.

**To Reproduce**
1. Set up a project for Docs
2. Don't include props for a component
3. See the "no components found" box

**Expected behavior**
"no component found"Box should not appear

**Screenshots**
![image](https://user-images.githubusercontent.com/29763590/61378252-fdd76280-a872-11e9-956c-164098c36b54.png)

**Comment by patdryburgh** [comment]
Hey @shilman, I happened to land here when trying to figure out how to add component props to my docs. Turns out because I am using `storiesOf`, I had to add `.addParameters({ component: Badge })` which was simple enough. 

Instead of removing this box, I wonder if providing a link to the documentation that explains the [Component parameter](https://github.com/storybookjs/storybook/blob/master/addons/docs/docs/docspage.md#component-parameter) would be more helpful. This would be similar to what `addon-knobs` does when there are no knobs configured for a story:

<img width="616" alt="Screen Shot 2019-11-13 at 10 46 04 AM" src="https://user-images.githubusercontent.com/751461/68793984-f3e82500-0602-11ea-9ddc-3b68ff8ec63e.png">

Thanks for all of the great work you and the Storybook team have put into this amazing tool. I've been a fan for a few years now and it just keeps getting better!
